views/employee/archive-department.php

```diff
@@ -16,6 +21,19 @@
 			?>
 		</header><!-- .page-header -->
 
+		<div>
+			<?php
+			// print custom field department, if it exists
+			if ( $fields->department ) : ?>
+				<h3><?php echo $fields->department; ?></h3>
+			<?php endif; ?>
+			<?php
+			// print custom field floor, if it exists
+			if ( $fields->floor ) : ?>
+				<h3><?php echo $fields->floor; ?></h3>
+			<?php endif; ?>
```

Comment:
I think for taxonomy archive more actual fields will be like some featured image and subheading.

fields like "floor" and "department" doesn't mean anything for grouped members page